research-software-directory/RSD-as-a-service
# rsd-admin can add/remove rsd-admin role to other users

# Assign rsd admin role

Changes proposed in this pull request:
*  RSD admin can assign/remove rsd-role to other rsd users using rsd users page in the administration section    
* Documentation page of rsd users is updated with new animation

How to test:
* `make start` to build solution and create test data
* login via SURFconext, as professor1 for example
* make professor1 user as rsd-admin via database update (see documentation)
* login via SURFconext as professor3 for example in order to create rsd account for the second user
* login as professor1 again and navigate to adminstration/rsd-users page
* search for professor3 by name `Newton` 
* assign professor3 role of rsd-admin by changing the switch near the name
* login as professor3 and confirm that it has rsd-admin role

## Change professor3 to rsd-admin

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/4e8a111c-1f7f-4118-87c9-a883c741eee5)

## Cannot change its own role
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/77ea24c7-c37d-4c77-8cb1-d1563b900f17)


PR Checklist:
